torch/ao/pruning/_experimental/pruner/README.md

```diff
@@ -1,93 +1,263 @@
-# Intro
+# Structured Pruning
+
+## Intro / Motivation
+
+**pruning** is the process of removing weights / connections in a neural network in order to make it run faster.
+
+There are two types of pruning that we support: **structured pruning** and **unstructured pruing**
+
+* **Unstructured pruning** refers to pruning any arbritary weight
+* **Structured pruning** refers to pruning by removing entire rows / columns of a weight.
+
+Structured pruning offers several advantages over unstructured pruning at the cost of lower granuality:
+1. We can store the masks more efficeintly in structured pruning.
+2. Structured pruning relies on removing columns / rows to a speedup whereas unstructured pruning requires custom kernels.
```

Comment:
could make this clearer to emphasize that structured pruning can use the same kernels as a normal, 

also

"to a speedup" should probably be "to obtain a speedup"

also, significantly less importantly, I think generally "speed-up" is used rather than "speedup"